incubator/hnc/Makefile

```diff
@@ -160,8 +163,6 @@ kind-deploy:
 	CONFIG=kind $(MAKE) deploy
 
 ###################### RELEASE ACTIONS #########################
-
-# Generate release YAML that can easily install HNC via `kubectl apply -f`
-release: manifests
-	mkdir -p releases
-	kustomize build config/default > releases/manifests.yaml
+# Build the release artifacts by Cloud Build
+release:
+	gcloud builds submit --config cloudbuild.yaml .
```

Comment:
Who 'owns' the official gcloud account for this? If I run this, it seems it'll use the current gcloud account I'm logged in on which is different from yours. Does this tie into the official k8s sig way of releasing (if there is one?).

Reply:
Thanks for pointing it out!

- I am trying to follow https://github.com/kubernetes/test-infra/blob/master/config/jobs/image-pushing/README.md here
- Currently I am playing around with it on a personal GCP project. It doesn't matter which `gcloud` account is used, as long as that it's built by Cloud Build (a trusted service that is transparent). Later manual work is required to download those artifacts and to upload the images to MTWG's Docker Hub and Github release page for YAMLs. (This is also included in the updated README.)

Hopefully it answers your questions.